<!-- -->
<launch>
 <include file="$(find tis_camera)/launch/$(env AGENT)/generic.launch"></include>

 <node name="omni_vision_node" pkg="omni_vision" type="omni_vision_node" output="screen"/>

</launch>

